.zuul.d/jobs.yaml

```
@@ -133,6 +133,56 @@
 
 # =============================================================================
 
+- job:
+    name: ansible-runner-build-container-image-stable-2.11
+    parent: ansible-build-container-image
+    provides:
+      - ansible-runner-stable-2.11-container-image
+    requires:
+      - python-base-container-image
+      - python-builder-container-image
+    required-projects:
+      - name: github.com/ansible/ansible
+        # NOTE (amhatre): Switch to stable-2.11 once branch is created.
+        override-checkout: devel
+    vars: &ansible_runner_image_vars_stable_2_11
+      container_images: &container_images_stable_2_11
+        - context: .
+          build_args:
+            - ANSIBLE_BRANCH=stable-2.11
+            # NOTE(pabelanger): There is no ansible-core 2.11, so we have to build ansible-base 2.11 ourself.
+            - ANSIBLE_CORE_IMAGE=quay.io/ansible/python-base:latest
```

this can be dropped

---

so we dont need  build_args key?
```suggestion
          build_args:
            - ANSIBLE_BRANCH=stable-2.11
            # NOTE(pabelanger): There is no ansible-core 2.11, so we have to build ansible-base 2.11 ourself.
            - ANSIBLE_CORE_IMAGE=quay.io/ansible/python-base:latest
```

---

right, we can drop them in favor of the required ansible-core image